internal/api/v1/models/app.go

```diff
@@ -62,6 +64,11 @@ func (ar *AppRef) EnvSecret() string {
 	return ar.Name + "-env"
 }
 
+func (ar *AppRef) PVCName() string {
+	// TODO: Can this exceed the allowed limit in characters?
+	return fmt.Sprintf("%s-%s", ar.Org, ar.Name)
```

Comment:
Irrelevant but since I remembered it here: The application image name doesn't have the org in it. This means the same application on different orgs could result in a conflict on the image name? We append more things to the image name so maybe that is impossible to happen, but I will check.

Reply:
IIRC
Application routes are already global.
Which means that you cannot have two applications with the same name, even in different orgs.

Reply:
true. I'm not sure if we have any friendly validation for this though. It probably allows the user to push with a non-unique name and fails miserably later, not sure.

Reply:
Note however, this apparently generates the name of a kube resource, i.e. of a `PVC`.
Might need a prefix. Will, if an org name like `12monkeys` is allowed.
Certainly needs `names.GenerateResourceName` from #700 to ensure the length limit.